app/helpers/common_parameters_helper.rb

```diff
@@ -29,9 +30,15 @@ def parameter_value_content(id, value, options)
     if lookup_key.present? && lookup_key.hidden_value?
       field = password_field_tag(id, value, option_hash)
     else
-      field = text_area_tag(id, value, option_hash)
+      case lookup_key_type
```

Comment:
Please extract this to a separate method, `lookup_key_field` or something similar